ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.7-presubmits.yaml

```diff
@@ -13,6 +13,7 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-network-operator-release-4.7-e2e-agnostic-upgrade
+    optional: true
```

Comment:
it was fixed to use origin/quay for 4.7 and later right? So should we just make them optional for 4.6?

Reply:
good point, reverting the others